app/controllers/sessions_controller.rb

```diff
@@ -1,14 +1,20 @@
 class SessionsController < ApplicationController
   skip_before_action :require_login
-  
+
   def create
-    if user = User.from_google_auth(request.env["omniauth.auth"])
+    user = User.from_google_auth(request.env["omniauth.auth"])
+    if user && user.role == "default"
```

Comment:
This and the logic check below could probably both be helper methods eg `is_default_user` `is_admin_user` or similar. Eventual possible refactor.